File: src/main/scala/com/socrata/controller/InfoController.java
package com.socrata.controller;

import com.socrata.api.InfoApi;
import com.socrata.models.InfoRequest;
import com.socrata.models.InfoResponse;
import io.smallrye.mutiny.Multi;
import jakarta.enterprise.context.ApplicationScoped;

@ApplicationScoped
public class InfoController implements InfoApi {
    @Override
    public Multi<InfoResponse> postInfo(InfoRequest infoRequest) {
        return Multi.createFrom().empty();
    }
}

File: server-redshift/src/main/scala/com/socrata/controller/InfoController.scala
package com.socrata.controller

import com.socrata.api.InfoApi
import com.socrata.model.{InfoRequest, InfoResponse}
import jakarta.enterprise.context.ApplicationScoped

import java.{lang, util}

@ApplicationScoped
class InfoController extends InfoApi {
  override def infoDatasetPost(dataset: String, infoRequest: InfoRequest, rowCount: Integer, copy: String, rollupName: String, obfuscatedId: lang.Boolean, queryTimeoutSeconds: Integer, xSocrataDebug: String, xSocrataAnalyze: String, context: String): util.List[InfoResponse] = ???
}
